inc/ticket.class.php

```diff
@@ -3708,11 +3708,10 @@ function showFormHelpdesk($ID, $ticket_template = false) {
          }
       }
 
-      // Get default values from posted values on reload form
```

Comment:
I guess some values will be lost on reload if you remove that

Reply:
This block is repeated
```php
      // Get default values from posted values on reload form
      if (!$ticket_template) {
         if (isset($_POST)) {
            $options = $_POST;
         }
      }
```

Reply:
Oh, you're right, I did not pay attention